cartographer_ros/cartographer_ros/dev/trajectory_comparison_main.cc

```diff
@@ -43,6 +44,8 @@ DEFINE_string(bag_filename, "",
 DEFINE_string(tf_fixed_frame, "map", "The name of the fixed frame.");
 DEFINE_string(cartographer_tracking_frame, "base_link",
               "The frame that cartographer is tracking.");
+DEFINE_double(skip_seconds, 60.0,
```

Comment:
Maybe default should be based on current functionality? That comparison starts from 0.0?

Reply:
True. Updated.